source/threads.tex

```diff
@@ -1207,6 +1207,9 @@
   // hash support
   template<class T> struct hash;
   template<> struct hash<thread::id>;
+
+  template<class charT>
+  struct formatter<thread::id, charT>;
```

Comment:
This is paper-correct. But I think it should be inserted after `operator<<`. That would be consistent with the other specialization of `formatter` added in this PR. It'd also be consistent with the insertion place of the detailed specifications.